frontend/integration-tests/tests/kubevirt/utils.ts

```diff
@@ -56,12 +56,24 @@ export function createResources(resources) {
 export function deleteResources(resources) {
   resources.forEach(resource => {
     const kind = resource.kind === 'NetworkAttachmentDefinition' ? 'net-attach-def' : resource.kind;
-    execSync(`kubectl delete -n ${resource.metadata.namespace} --cascade ${kind} ${resource.metadata.name}`);
+    let namespace = '';
```

Comment:
@suomiy @gouyang  isn't there a better way to solve this? These delete/create method were intended for manifest objects, which have of course structure `kind, metadata.name, metadata.namespace`, not for class instances, at least with their current implementation.
- Isn't a better solution to provide our classes with metadata property that would contain name and namespace? Instead of adding complicated logic?
- Or maybe we distinguish between resource manifest and class instance so that if it's a class instance we would call `instance.delete()` class method that would handle the deletion.

Reply:
I guess you are right. It is better to have united interface.

I propose to add this method to `DetailView`:

```js
  asResource() {
    return {
      kind: this.kind,
      metadata: {
        namespace: this.namespace,
        name :this.name,
      },
    };
  }
```

Reply:
Even we have another method for the class to delete the vm instance, it need separate lines of code to delete the resources like testNad, vm. How about just use `execSync`, it's quite directly and simple to read.  

Let's get it out of the scope of this PR, I added `vm.action('Delete')` instead of deleting the vm in `deleteResources`.

Reply:
> Even we have another method for the class to delete the vm instance, it need separate lines of code to delete the resources like testNad, vm. How about just use execSync, it's quite directly and simple to read.

it is prone to mistakes and having an abstraction for simple things like CRUD is easier to read IMO.

I think, it is not necessary to delete VMs in UI when testing nics.

@rhrazdil your opinion?

Reply:
@gouyang @suomiy I tend to agree, this is a small change that will allow removing created resources in a uniformed fashion. Also, this way we'll reduce execution time without losing coverage.

Reply:
Done by add `asResource` into `DetailView`.